src/styles/core/typography/_font-styles.scss

```diff
@@ -69,7 +69,7 @@ $font-styles: (
     large: 14_20
   ),
   body-reduced-tight: (
-    large: 14_18
+    large: 17_25
```

Comment:
This is used in a number of places outside of the localnav/navigator. Are you sure we want to be updating this instead of creating a new style?

Reply:
You are right, it was affecting more places. I created new font styles, also for the nav title in documentation: https://github.com/apple/swift-docc-render/pull/772/commits/fa617e839c68f4023ff4055493d850d416734670